<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDTTrainingSampleDockWidget</class>
 <widget class="QDockWidget" name="CDTTrainingSampleDockWidget">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Training Samples</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBoxSamples">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Training Samples</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QFrame" name="frameSample">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>3</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButtonEditSample">
              <property name="toolTip">
               <string>Edit Samples</string>
              </property>
              <property name="text">
               <string>Edit Samples</string>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/Icon/Edit.png</normaloff>:/Icon/Edit.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonSampleRename">
              <property name="toolTip">
               <string>Rename</string>
              </property>
              <property name="text">
               <string>Rename</string>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/Icon/Rename.png</normaloff>:/Icon/Rename.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonNewSample">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Insert New Sample</string>
              </property>
              <property name="text">
               <string>Insert New Sample</string>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/Icon/Add.png</normaloff>:/Icon/Add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonRemoveSelected">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Remove Selected Sample</string>
              </property>
              <property name="text">
               <string>Remove Selected Sample</string>
              </property>
              <property name="icon">
               <iconset resource="../../resource.qrc">
                <normaloff>:/Icon/Minus.png</normaloff>:/Icon/Minus.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="listView"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
